### PR TITLE
Show the workflow state for all authenticated users

### PR DESCRIPTION
**Pull request by tiberiuichim**
This is just to enable comparison to the #494 PR

### CODE REVIEW

File: kotti/views/edit/actions.py [hunk 1]
```diff
@@ -162,6 +162,7 @@ def paste_nodes(self):
             else:
                 self.flash(_(u'Could not paste node. It no longer exists.'),
                            'error')
+        DBSession.flush()
```
Comment (disko) [inline kotti/views/edit/actions.py]: How is this related to the issue?

Comment (tiberiuichim) [inline kotti/views/edit/actions.py]: Without, there's a strange error in test_functional
